### DISCUSSION TITLE
Captioning videos

### DISCUSSION THREAD

**Discussion by heaversm** [mozilla/hubs#3476] [Spoke Questions] [answered]
Curious how folks have been captioning videos - do I need to bake these into my videos and export? Is there a way to utilize youtube videos and auto-captioning? Does hubs / spoke itself have a way to address this (or any plans in the works)?

**Comment by matthewbcool** [comment, marked as answer]
Seems like hard coded subtitles would be the best way to go to me. For a live event you can stream your audio through webcaptioner.com and screenshare that into the room but results are not always great. Linking with youtube is not recommended.

**Reply by heaversm** [reply]
Thanks - web captioner is a good resource. Can you say more about why youtube is not recommended? Seems like even if you give a youtube URL, you don't get a youtube player or the option to toggle captions, so I guess it's sort of pointless...

**Reply by cvanmeurs** [reply]
YouTube frequently blocks videos from being streamed into a Hubs room.  Sometimes it works, but more often than not, they won't play.  It's annoying.

**Reply by heaversm** [reply]
I ended up finding a workflow that works pretty good for me as far as baking-in the captions using youtube's auto-captioning and handbrake. [I documented it here](https://youtu.be/1BiJH61FKCQ).

**Reply by curtgrimes** [reply]
Web Captioner will sunset on October 31, 2023. If you are still using it today, please consider alternatives. More info: https://webcaptioner.com
